examples/app-with-scenes/README.md

```diff
@@ -1,14 +1,14 @@
-# Grafana Scenes App Plugin Template
+# Grafana Scenes App Plugin example
 
 This example demonstrates how to build a basic app plugin using @grafana/scenes framework.
 
 ## What are Grafana app plugins?
 
```

Comment:
it may be worth it to review the original template too, for example: https://github.com/grafana/plugin-tools/blob/main/packages/create-plugin/templates/app/README.md?plain=1#L7 (which is using `datasources`, not  `data sources`).

Reply:
done.